.github/workflows/ci_cron_weekly.yml

```diff
@@ -29,10 +29,10 @@ jobs:
           # that gives too many false positives due to URL timeouts. We also
           # install all dependencies via pip here so we pick up the latest
           # releases.
-          - name: Python 3.8 with dev version of key dependencies
+          - name: Python 3.10 with dev version of key dependencies
             os: ubuntu-latest
-            python: 3.8
-            toxenv: py38-test-devdeps
+            python: 3.10.0-rc.1
```

Comment:
I think there is a syntax that allows you to say 'or later 3.10 releases'? If you can't find it I can look when I am back at a computer today.

Reply:
´´´
python-version: '3.10.0-alpha - 3.10.0 # SemVer's version range syntax
´´´

Reply:
Done, thanks!